Firmware/RTK_Surveyor/Begin.ino

```diff
@@ -29,14 +27,89 @@ void beginBoard()
   {
     productVariant = REFERENCE_STATION;
   }
-  else if (isConnected(0x19) == true) //Check for accelerometer
+  else
   {
-    if (zedModuleType == PLATFORM_F9P) productVariant = RTK_EXPRESS;
-    else if (zedModuleType == PLATFORM_F9R) productVariant = RTK_EXPRESS_PLUS;
+    productVariant = RTK_UNKNOWN; // Need to wait until the GNSS and Accel have been initialized
   }
-  else
+}
+
+void initializePowerPins()
```

Comment:
This feels like it should go inside initializeBoard() with the other platforms. Why separate this chunk and not the other platforms? I don't see many uniquely power pins. Looks like all the other platform inits.

Reply:
The Reference Station is (probably) unique in that you need to enable power for (e.g.) the Display before you can talk to it.
`initializePowerPins` allows you to turn anything on that needs to be turned on before you `beginGNSS`, `beginAccel` or `beginDisplay`.